src/common/transformations/src/ngraph_ops/augru_sequence.cpp

```diff
@@ -0,0 +1,180 @@
+                              Dimension::merge(merged_num_directions, merged_num_directions, w_pshape[0]) &&
+                              Dimension::merge(merged_num_directions, merged_num_directions, r_pshape[0]) &&
+                              Dimension::merge(merged_num_directions, merged_num_directions, b_pshape[0]),
+                          "Parameter num_directions not matched in AUGRUSequence.");
+
+    auto valid_num_directions = 1;  // AUGRUSequence supports only Forward direction
+    NODE_VALIDATION_CHECK(this,
+                          Dimension::merge(merged_num_directions, merged_num_directions, valid_num_directions),
+                          "Parameter 'num_directions' doesn't match with direction '",
+                          m_direction,
+                          "' in AUGRUSequence. Expected ",
+                          valid_num_directions,
+                          ", actual ",
+                          merged_num_directions);
+
+    // Validate hidden_size value for W, R, B inputs
+    if (merged_hidden_size.is_static()) {
+        if (w_pshape[1].is_static()) {
+            NODE_VALIDATION_CHECK(this,
+                                  w_pshape[1].compatible(merged_hidden_size * gru_seq_gates_count),
+                                  "Parameter hidden_size mistmatched in W input. Current value is: ",
+                                  w_pshape[1].get_length(),
+                                  ", expected: ",
+                                  merged_hidden_size.get_length() * gru_seq_gates_count,
+                                  ".");
+        }
+
+        if (r_pshape[1].is_static()) {
+            NODE_VALIDATION_CHECK(this,
+                                  r_pshape[1].compatible(merged_hidden_size * gru_seq_gates_count),
+                                  "Parameter hidden_size mistmatched in R input. Current value is: ",
+                                  r_pshape[1].get_length(),
+                                  ", expected: ",
+                                  merged_hidden_size.get_length() * gru_seq_gates_count,
+                                  ".");
+        }
+
+        if (b_pshape[1].is_static()) {
+            NODE_VALIDATION_CHECK(
+                this,
+                b_pshape[1].compatible(merged_hidden_size *
+                                       (m_linear_before_reset ? (gru_seq_gates_count + 1) : gru_seq_gates_count)),
+                "Parameter hidden_size mistmatched in B input. Current value is: ",
+                b_pshape[1].get_length(),
+                ", expected: ",
+                merged_hidden_size.get_length() *
+                    (m_linear_before_reset ? (gru_seq_gates_count + 1) : gru_seq_gates_count),
+                ".");
+        }
+    }
+
+    // Mark inputs which are relevant to output parameters
+    for (size_t i = 0; i <= 6; ++i) {
+        set_input_is_relevant_to_shape(i);
+    }
```

Comment:
Seems they are not relevant. We set it for the inputs that have data which must be used for shape calculation.
Like second input of Reduce -- this is relevant because it's an axis and without it we wouldn't be able to calculate output shape.
![image](https://user-images.githubusercontent.com/55839243/186384119-a63a097d-bb5b-4f31-b209-71d2f9c35df6.png)

Reply:
Done. In general it is not obious to say which inputs must be provide to shape calculation, because in AUGRUCell/AUGRUSequence case we can read needed dimnesions from different inputs. I've changed it to set minimal set of inputs as relevant.